pkgs/applications/misc/haunt/default.nix

```diff
@@ -27,10 +27,22 @@ stdenv.mkDerivation rec {
     guile-reader
   ];
 
-  postInstall = ''
-    wrapProgram $out/bin/haunt \
-      --prefix GUILE_LOAD_PATH : "$out/share/guile/site:${guile-commonmark}/share/guile/site:${guile-reader}/share/guile/site" \
```

Comment:
Why did you wipe out guile-commonmark and guile-reader here?

Reply:
The build inputs are added to the load paths by [Guile's setup hook](https://github.com/NixOS/nixpkgs/blob/bd64a6221f18a24c79b879c0441f4420f0f2d47e/pkgs/development/interpreters/guile/setup-hook-2.2.sh), so we can just prepend Haunt's path to those.

Reply:
Oh, good! I will try the same in skribilo then!